packages/pipelines/src/commands/reviewapps/disable.ts

```diff
@@ -0,0 +1,58 @@
+import {Command, flags} from '@heroku-cli/command'
```

Comment:
I often, for clarity, do `import {Command, flags as HerokuFlags} from '@heroku-cli/command'`.